data_steward/cdr_cleaner/cleaning_rules/deid/missing_and_replaced_concept_suppression.py

```diff
@@ -0,0 +1,88 @@
+"""
+Ensures that all the replaced concepts in vocabulary are being suppressed in Controlled tier datasets
+
+
+Original Issue: DC-2278
+
+The intent of this cleaning rule is ensure the concepts listed for suppression in the controlled tier are
+ sandboxed and then suppressed.
+"""
+
+# Python imports
+import logging
+
+# Third party imports
+from pandas import read_csv
+
+# Project imports
+from resources import MISSING_PRIVACY_STANDARD_CONCEPTS_PATH, REPLACED_PRIVACY_CONCEPTS_PATH
+from common import CDM_TABLES
+from utils import pipeline_logging
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.deid.concept_suppression import \
+    AbstractInMemoryLookupTableConceptSuppression
+
+LOGGER = logging.getLogger(__name__)
+ISSUE_NUMBERS = ['DC2278']
+
+
+class ControlledTierReplacedConceptSuppression(
+        AbstractInMemoryLookupTableConceptSuppression):
```

Comment:
I think this may be ok for now.  I don't think it will hit any quota limits.  But if the list were to expand, it could.  That is why I was suggesting to extend the `AbstractBqLookupTableConceptSuppression` class.  You don't have to create the table via a query. You could load it via the `create_suppression_lookup_table` method since it has access to a client object.  But, this is an optional change.

Reply:
you are right, updated CR to store the concept_ids in the table.